src/FCMClient/JSON/Types.hs

```diff
@@ -198,7 +203,7 @@ $(deriveJSON (aesonPrefix snakeCase) { omitNothingFields = True } ''FCMNotificat
 
 
 instance Default FCMNotification where
-  def = FCMNotification Nothing Nothing Nothing Nothing Nothing Nothing Nothing Nothing Nothing Nothing Nothing Nothing Nothing
+  def = FCMNotification Nothing Nothing Nothing Nothing Nothing Nothing Nothing Nothing Nothing Nothing Nothing Nothing Nothing Nothing
```

Comment:
Can you initialize this using the record names instead? Makes it more obvious which field is what and it should prevent any errors down the road if we need to change any of the values in the middle. Lets also confirm that Werror is enabled so that we can catch any uninitialized fields down the road

Reply:
Done. The latest changes also involves changes by stylish haskell so the diff might be a little too much I guess.